src/nl/rubensten/texifyidea/settings/LabelingCommandInformation.kt

```diff
@@ -0,0 +1,23 @@
+package nl.rubensten.texifyidea.settings
+
+data class LabelingCommandInformation(var commandName: String, var position: Int, var labelPrevCmd: Boolean) {
```

Comment:
`labelPrevCmd` -> `labelsPreviousCommand`.

Reply:
fixed in commit ed34f48cd6dd8da4a8ed3d61d6cf44599f4bc7dd